README.md

```diff
@@ -59,6 +59,5 @@ Promise.race([
 
 ## Unhandled Rejections
 
-In node lie emits `unhandledRejection` events when promises are not handled in
-line with how [iojs does so](https://iojs.org/api/process.html#process_event_unhandledrejection)
-meaning it can act as promise shim in node as well as the browser.
+
+lie emits an `unhandledRejection` event when a rejected promise isn't caught, in line with [how iojs does it](https://iojs.org/api/process.html#process_event_unhandledrejection). This allows it to act as a promise shim in both node and the browser.
```

Comment:
The original text did specify it was for node. I don't recall specifically why I removed it; probably because I assumed (incorrectly) that it was unnecessary. In any case, I re-added the clarification.

Also, I rephrased node -> Node.js as per their [https://nodejs.org/static/documents/trademark-policy.pdf](trademark policy).